component/client/core/ClientCore.tsx

```diff
@@ -101,8 +101,8 @@ class ClientCore extends React.Component<any, ClientCoreState> {
 
   private handleSave(): void {
     storage.set(STORAGE_KEY_USERNAME, this.state.username)
-    .then(storage.set(STORAGE_KEY_INITIALIZED, true)
-      .then(this.setState({initialized: true, username: ""})));
+      .then(() => storage.set(STORAGE_KEY_INITIALIZED, "true")
```

Comment:
Why do you need to change booleans to strings? Chrome seems to serialize and unserialize them just fine.
Also, indents should be aligned. Either change it to align all the dots or align on the left edge.